client/asset/eth/eth.go

```diff
@@ -4695,54 +4733,39 @@ func (w *baseWallet) checkPendingTxs() {
 
 const txHistoryNonceKey = "Nonce"
 
-func (w *baseWallet) addToTxHistory(nonce uint64, balanceDelta int64, fees, blockNumber uint64,
-	assetID uint32, id dex.Bytes, typ asset.TransactionType) {
-	w.tipMtx.RLock()
-	tip := w.currentTip.Number.Uint64()
-	w.tipMtx.RUnlock()
-	var confs uint64
-	if blockNumber > 0 && tip >= blockNumber {
-		confs = tip - blockNumber + 1
-	}
-
+func (w *baseWallet) addToTxHistory(nonce, amount, fees uint64, assetID uint32, txHash common.Hash, typ asset.TransactionType, recipient *string) {
```

Comment:
Nit: typ -> type

Reply:
`type` is a reserved word. I'll change to `txType`.